framework/source/class/qx/ui/layout/VBox.js

```diff
@@ -521,12 +521,12 @@ qx.Class.define("qx.ui.layout.VBox",
       minHeight += percentMinHeight;
 
       // Respect gaps
-      var spacing = this.getSpacing();
+      var spacing = this.getSpacing(), gaps;
```

Comment:
```suggestion
      var spacing = this.getSpacing();
      var gaps;
```

Reply:
@lucapivato : yes please add this change and put `var gaps;` on a separate line.